src/components/infinite-scroll/InfiniteScroll.tsx

```diff
@@ -93,6 +93,9 @@ function InfiniteScroll({
             container.addEventListener('resize', resizeHandler);
         }
 
+        // calls onContainerScroll until page becomes scrollable
+        onContainerScroll(params);
```

Comment:
we don't want to throttle this, because if useEffect gets triggered again, and this doesn't get called due to a throttle, it will stop loading

Reply:
what about a debounce function? would that be appropriate instead of a throttle?
https://lodash.com/docs/4.17.15#debounce

Reply:
I think in general it doesn't make sense to throttle this one, since it only get's called once per useeffect, which only happens upon prop changes. In case data fetches very fast, this function being throttled means we stop fetching. It's a bit hard to explain, I'm happy to hop on a call regarding this tomorrow